isites_migration/utils.py

```diff
@@ -82,6 +82,20 @@ def export_files(keyword):
                 file_path = os.path.join(root, file)
                 z_file.write(file_path, file_path[zip_path_index:])
         z_file.close()
+
+        # we want to log the size of the data we exporting per TLT-2099
+        logger.info('Creating zip file %s' % zip_filename)
```

Comment:
Why are we closing and then reopening the zipfile?  We should be able to ask for the `ZipInfo` for every file as we write it to the zipfile in the loop above this, no?  ie., right after the `z_file.write(...)` call, just do